src/styles/styles.js

```diff
@@ -638,14 +638,14 @@ const styles = {
     },
 
     visuallyHidden: {
-        ...visibility('hidden'),
+        ...visibility.hidden,
         overflow: 'hidden',
         width: 0,
         height: 0,
     },
 
     visibilityHidden: {
-        ...visibility('hidden'),
+        ...visibility.hidden,
```

Comment:
@fabioh8010 I'm assuming it didn't work before, and no styles were spread here? Let's double check that this change doesn't break anything 😄

Reply:
@blazejkustra this `visibility` is only being used inside `styles.js` and had a very odd implementation compared to other utilities, that's why I changed to this way.

Reply:
Ahh I see, I didn't check how it was implemented before. LGTM